docs/src/pages/components/badges/AccessibleBadges.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Badge from '@material-ui/core/Badge';
+import MailIcon from '@material-ui/icons/Mail';
+
+function notificationsLabel(count) {
+  if (count === 0) {
+    return 'no notifications';
+  }
+  if (count > 99) {
+    return 'more than 99 notifications';
+  }
+  return `${count} notifications`;
+}
+
+const preventDefault = (event) => event.preventDefault();
+
+export default function AccessibleBadges() {
+  return (
+    <IconButton
+      aria-label={notificationsLabel(100)}
+      href="/inbox"
+      onClick={preventDefault}
```

Comment:
What is this supposed to do?

Reply:
Prevent to lose the context of the developers that click on the link. We can find this approach in https://next.material-ui.com/components/links/#basic-links

Reply:
Then let's not use a link here. Demos are intended to be c&p.  This isn't it.

Reply:
Fair, we don't need a link to make our point.